### PR TITLE
Resolvers: Create tests for posts.js

### LINKED ISSUE #589: Resolvers: Create tests for posts.js
The Talawa-API code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/resolvers/post_query/posts.js`

- We will need the API to be refactored for all methods, classes and/or functions found in this file for testing to be correctly executed.
- When complete, all all methods, classes and/or functions in the refactored file will need to be tested. These tests must be placed in a
single file with the name `tests/resolvers/post_query/posts.spec.js`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa-api/issues/490

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa-api/tree/develop/lib/resolvers/post_query/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.

### PR DESCRIPTION
**Pull request by priyang12**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #589

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Your tests are failing. Please fix.

**Comment by priyang12** [comment]
@palisadoes looks like the latest Organisation query is failing in my workflow?

**Comment by palisadoes** [comment]
Please try to fix it. The array's order seems to be sorted differently from expected. It seems simple to do.

**Comment by JamaicanFriedChicken** [comment]
Can you try to get the code coverage to 100%? @priyang12

**Comment by priyang12** [comment]
@JamaicanFriedChicken hi I have added new tests for likes and comments and the coverage is 100 can please take a look at it.

**Comment by palisadoes** [comment]
Please fix the failing tests:
- https://github.com/PalisadoesFoundation/talawa-api/runs/5693546529?check_suite_focus=true

File: tests/resolvers/post_query/posts.spec.js [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    const args = {
+      orderBy: 'commentCount_ASC',
+    };
+    const response = await GetPosts({}, args, { userId: UserId });
+    // Sort the posts by comments
+    const sortedPosts = Posts.sort((a, b) => {
+      return a.comments - b.comments;
+    });
+    console.log(sortedPosts[0].comments.length);
+    console.log(response[0].comments.length);
```
Comment (JamaicanFriedChicken) [inline tests/resolvers/post_query/posts.spec.js:249]: remove the console.logs() here.

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]

**Pull request closed without merging**